### PR TITLE
added repositories in suggestion

### PR DESCRIPTION
**Pull request by himorin**
Added six repositories to list of activities and repositories.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -169,6 +170,7 @@ <h3>Recommendation Track specification repositories</h3>
         <li><a href="https://github.com/w3c/ttml3/pulse">TTML3</a></li>
         <li><a href="https://github.com/w3c/webvtt/pulse">WebVTT</a></li>
         <li><a href="https://github.com/w3c/adpt/pulse">ADPT</a></li>
+        <li><a href="https://github.com/w3c/tt-module-registry/pulse">TTML Extension Module Registry</a></li>
```
Comment (cconcolato) [inline index.html]: I think the group decided not to move forward with this registry. I'm not sure it's a good idea to advertise it on the group's page.

File: index.html [hunk 2]
```diff
@@ -178,8 +180,12 @@ <h3>Recommendation Track specification repositories</h3>
       <h3>Test repositories</h3>
       <ul>
         <li><a href="https://github.com/w3c/imsc-tests/pulse">IMSC Tests</a></li>
+        <li><a href="https://github.com/w3c/IMSC-1.0.1_Text_TestContent/pulse">Test contents for IMSC 1.0.1 Text</a></li>
+        <li><a href="https://github.com/w3c/IMSC-1.1_Text_TestContent/pulse">Test contents for IMSC 1.1 Text</a></li>
+        <li><a href="https://github.com/w3c/IMSC-1.1_Image_TestContent/pulse">Test contents for IMSC 1.1 Image</a></li>
         <li><a href="https://github.com/w3c/ttml1-tests/pulse">TTML1 tests</a></li>
         <li><a href="https://github.com/w3c/ttml2-tests/pulse">TTML2 tests</a></li>
+        <li><a href="https://github.com/w3c/ttml3-tests/pulse">TTML3 tests</a></li>
```
Comment (cconcolato) [inline index.html]: I think a goal for TPAC would be to decide what to do with TTML3. So far, it seems that the 2nd edition of TTML2, the extension modules and the new version of IMSC would be sufficient.

File: index.html (hunk 1 above)
Comment (nigelmegitt) [reply index.html]: I agree, we should not add it at this time.

File: index.html [hunk 3]
```diff
@@ -160,6 +160,7 @@ <h3>Group repositories</h3>
       <ul>
         <li><a href="https://github.com/w3c/ttwg/pulse">TTWG business</a></li>
         <li><a href="https://github.com/w3c/tt-reqs/pulse">TTWG future requirements</a></li>
+        <li><a href="https://github.com/w3c/charter-timed-text/pulse">Draft charter for the Timed Text Working Group</a></li>
```
Comment (nigelmegitt) [inline index.html]: I don't think we would normally advertise a repo whose only purpose is for changing the charter, which happens only every couple of years or so.

Comment (himorin) [reply index.html]: removed

File: index.html (hunk 2 above)
Comment (himorin) [reply index.html]: removed per TPAC discussion
> in terms of goals, we decided not to progress TTML3 for the time being

File: index.html (hunk 1 above)
Comment (himorin) [reply index.html]: removed

**Comment by himorin** [comment]
updated with removing three lines (links to repos).

**Pull request merged**